.travis.yml

```diff
@@ -17,13 +16,9 @@ python:
 
 env:
   global:
-  - VERSION="12.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
-  - TRANSIFEX_USER='transbot@odoo-community.org'
-  - secure: MZ5W99aYtT5mntw9Aur7bZ0FFXHZYc3v2oWg9dEODeKcLWUTmQbPC/T/pcjex1MBQTIHy0GZ+M/kj7qQQilbtbMYnjt7VBcY+fY1cZskEohc7mK/c8sUNHeUYj7Ec7n7VriY0XjQVH7Ov3FkiDoUm/4um0VyhH8LpKWw0vMhqFY=
-  
+  - VERSION="12.0" TESTS="0" LINT_CHECK="0"
```

Comment:
@astirpe, I think it should be better move the changes related to `TRANSIFEX` onto a separate PR.
While that one related to ` postgresql` is ok as it's specific to the current module.

Thank you!

Reply:
@tafaru I believe that the postgres 9.2 does not fit with Odoo 12.0, it seems that the minimum required version of postgres is 9.5. So IMO we could create one separate PR for both postgres version and transifex removal.

I'm at the moment busy at the Odoo Experience. If you want and can, you could open the PR and I will approve it quickly. Otherwise I will do asap.

Thank you

Reply:
@astirpe, I resolve this conversation as https://github.com/OCA/account-financial-reporting/pull/468#issuecomment-426932450
Enjoy your Odoo Experience!